hack/validate-dashboard.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+#
+# Copyright 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+echo "> Validating dashboards"
+
+
+cd "$(dirname "$0")/../test/validate-dashboard"
+
+# Prompt the user for the path to the Grafana dashboard JSON file
+read -p "Enter the path to the Grafana dashboard JSON file: " jsonFilePath
```

Comment:
Why not checking all the paths? Why do we need user input?

Reply:
> Why not checking all the paths? Why do we need user input?

Yeah, it's not a good idea to ask user input. Updated the code to check the path of dashboards but not all the paths as the other json files are not with same content structure like have the title.